### PR TITLE
feat(route): add minecraft news

### PR DESCRIPTION
**Pull request by OutlinedArc217**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
minecraft/news
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [X] New Route / 新的路由
  - [X] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [X] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [X] Parsed / 可以解析
  - [X] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/minecraft/news.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['minecraft.net/'],
+        },
+    ],
+    name: 'Minecraft News',
```
Comment (TonyRL) [inline lib/routes/minecraft/news.ts:23]: No need to include namespace name https://github.com/DIYgod/RSSHub/blob/5a188a3c5c949dc66802529fcad0291d7ef34550/lib/routes/minecraft/namespace.ts#L4 again in route name.

File: lib/routes/minecraft/news.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['minecraft.net/'],
```
Comment (TonyRL) [inline lib/routes/minecraft/news.ts:20]:
```suggestion
            source: ['minecraft.net/en-us/articles', 'minecraft.net/'],
```

File: lib/routes/minecraft/news.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        },
+    ],
+    name: 'Minecraft News',
+    maintainers: ['OutlinedArc217'],
+    handler,
+    url: 'minecraft.net/',
+    description: `Catch up on the latest articles`,
```
Comment (TonyRL) [inline lib/routes/minecraft/news.ts:27]:
```suggestion
    description: 'Catch up on the latest articles',
```

File: lib/routes/minecraft/news.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+    return {
+        title: 'Minecraft News',
+        link: articlesUrl,
+        description: 'Catch up on the latest articles',
+        item: detailedItems.map((item) => ({
+            title: item.title,
+            link: item.link,
+            description: item.description,
+        })),
```
Comment (TonyRL) [inline lib/routes/minecraft/news.ts:71]: Unnecessary map()

**Comment by OutlinedArc217** [comment]
@TonyRL done.

**Review by TonyRL — COMMENTED** [review]
Route tests are not passing at all. https://github.com/DIYgod/RSSHub/pull/17763#discussion_r1866006672 and https://github.com/DIYgod/RSSHub/pull/17763#discussion_r1866010632 are not resolved

**Comment by OutlinedArc217** [comment]
> Route tests are not passing at all. [#17763 (comment)](https://github.com/DIYgod/RSSHub/pull/17763#discussion_r1866006672) and [#17763 (comment)](https://github.com/DIYgod/RSSHub/pull/17763#discussion_r1866010632) are not resolved

[Namespace problem] (https://github.com/DIYgod/RSSHub/pull/17763#discussion_r1866010632) has already resolved in previous commit.

**Pull request closed without merging**